app/ts/background/popupMessageHandlers.ts

```diff
@@ -333,7 +338,7 @@ export async function homeOpened(simulator: Simulator) {
 			tabId,
 			rpcEntries: await getRpcList(),
 		}
-	})
+	}) as PartiallyParsedPersonalSignRequest)
```

Comment:
Figure out if there's a better way to do this

Reply:
I don't understand this, `PartiallyParsedPersonalSignRequest` and `UpdateHomePage` have different method's, and some of the expected `UpdateHomePage` values in the data.simulation don't match the  object.

Might need to use one type only and make it part of the ``MessageToPopup`` union?

Reply:
good catch, I fix that here: https://github.com/DarkFlorist/TheInterceptor/pull/669/files#diff-f3d3b02da582a58752f45484338f583264b08a4705fb534f7ef5c87666d24cc0R321